controllers/Payment/paymentController.js

```diff
@@ -0,0 +1,82 @@
+const { initializePayment, verifyPayment } = require('./Paystack')
+const { errorResMsg, successResMsg } = require('../../utils/libs/response')
+const orderModel = require('../../models/Payment/orderModel')
+const User = require('../../models/User/userModel.js')
+const Service = require('../../models/service.model')
+const _ = require('lodash')
+const dotenv = require('dotenv')
+const request = require('request')
+dotenv.config()
+
+// Using Paystack
+const paystackPayment = async (req, res) => {
+    const user = req.user
+    if(!user) return errorResMsg(res, 400, { message: "User not found" })
+    const findUser = await User.findById(user._id)
+    if(!findUser) return errorResMsg(res, 400, { message: "User profile not found" })
+    const findServiceOrder = await Service.findOne({ _id: serviceId })
+    if(!findServiceOrder) return errorResMsg(res, 400, { message: "Please include a service order Id" })
+    const form = _.pick(req.body,['amount','email','fullname']);
+    form.metadata = {
+        fullname : form.fullname
+    }
+    form.amount *= 100
+    const options = {
+        url: 'https://api.paystack.co/transaction/initialize',
+        headers: {
+            authorization: `${process.env.PaystackSK}`,
+            "content-type": "application/json",
+            "cache-control": "no-cache"
+        },
+        form  
+    }
+    request.post(options, async (error, response, body) => {
+        if(error) return errorResMsg(res, 400, { message: error })
```

Comment:
Alright Why not use axios instead of request npm package I think best practices you should use the axios or fetch npm package.